client/src/GlobalStyles.js

```diff
@@ -67,33 +67,55 @@ export const GlobalStyles = createGlobalStyle`
   max-width: 500px;
   margin: 50px auto;
   padding: 40px 25px;
-  background-color: whitesmoke;
-  font-family: Arial;
-  font-size: 16px;
+  //background-color: whitesmoke;
+  //font-family: Arial;
+  //font-size: 16px;
```

Comment:
If you don't need the styles anymore delete it! Commenting out code and pushing it into main just clutters up your codebase. If you want to bring it back you can always check your git history